### PR TITLE
Release 1.2.0 - Sprint 1

### PR DESCRIPTION
**Pull request by Im0tec**
# 🎬 Filmaro 🎬 - Pull Request

## Description:

Sprint closing, things worked:
#### 🧰Features:
- Pagination.
- Streaming page.
- Streamings dropdown.
#### 🐛Bug Fix:
- Mobile dropdown

## Type of change:

- [x] New feature (This PR adds a new functionality to the project)
- [ ] Technical debt (This PR improve features already created)
- [x] Bug fix (This PR covers solving a bug)
- [ ] Documentation (This PR require a documentation update, i.e. the class doc)
- [x] Issue resolution (This PR solve an github open issue)

## Checklist:

Delete this message and the options that is unnecessary, i.e. the options that are no relate to the changes you made.

- [x] I created a responsive component.
- [x] I followed the concept of clean code.
- [x] My changes do not generate a new warning or issue.
- [x] I deleted all the code lines that are not being used.
- [x] I commented my code in some areas i believed that are difficult to understand.
- [x] I have made corresponding changes to the documentation.
- [x] I create a release tag in case my changes is a breaking-change.

👩🏽‍💻~ Filmaro Team ~👨🏽‍💻

### CODE REVIEW

**Review by MatheusFrate — APPROVED** [review]

File: src/components/Homepage/Home/style.css [hunk 1]
```diff
@@ -59,4 +55,13 @@
...
\ No newline at end of file
+}
+
+@media screen and (max-width: 600px) {
+    .Movies-Series-Text {
+      flex: 50%;
+      position: relative;
+      left: 0;
+      text-align-last: center;
+    }
```
Comment (vitorSanchess) [inline src/components/Homepage/Home/style.css:66]: oia o home fazendo media query 🙏🏻🙏🏻🙏🏻

Comment (JoaoWudarski) [reply src/components/Homepage/Home/style.css:66]: 🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻

**Review by vitorSanchess — COMMENTED** [review]
aprovado meu consagrado

File: src/components/Homepage/Home/style.css (hunk 1 above)
Comment (Im0tec) [reply src/components/Homepage/Home/style.css:66]: 🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻

**Review by vitorSanchess — APPROVED** [review]
Aprovado meu consagrado
WhatsApp_1ONJ0WxwZJ

**Pull request merged**